airflow/providers/openlineage/extractors/__init__.py

```diff
@@ -20,4 +20,8 @@
 from airflow.providers.openlineage.extractors.base import BaseExtractor, OperatorLineage
 from airflow.providers.openlineage.extractors.manager import ExtractorManager
 
+"""
+:meta private:
+"""
```

Comment:
Why is the `__init__` file marked as private? Do we not want people to import from this file (but submodules instead?) and in that case, why are the classes re-exported from submodules into this file?

Reply:
Because this is just temporary. The idea was that first extractors will be added "as is" and then they will be merged into operators. This is what @mobuchowski plan is. We will not release openlineage provider (it is currently excluded from release) until this refactoring is done.

Reply:
The question then becomes why the imports are added to this file.